backend/src/main/java/gov/cdc/usds/simplereport/service/DeviceTypeService.java

```diff
@@ -189,6 +230,229 @@ public DeviceType createDeviceType(CreateDeviceType createDevice) {
+      lockAtMostFor = "PT30M")
+  public void scheduledSyncDevices() {
+    if (deviceSyncEnabled) {
+      syncDevices();
+    }
+  }
+
+  @Transactional
+  @AuthorizationConfiguration.RequireGlobalAdminUser
+  public void syncDevices() {
+    List<LIVDResponse> devices = client.getLIVDTable();
+
+    var devicesToSync = new HashMap<DeviceType, ArrayList<SupportedDiseaseTestPerformedInput>>();
+    var deviceSpecimens = new HashMap<String, List<UUID>>();
+
+    devices.forEach(
+        device -> {
+          // Skip sync for any device entries with incomplete data
+          if (device == null
+              || device.getManufacturer() == null
+              || device.getModel() == null
+              || device.getVendorAnalyteName() == null) {
+            log.info("One or more required values are not present in the LIVD entry");
+            log.info("Skipping device...");
+            return;
+          }
+
+          // Does the device exist at all in the DB?
+          log.info("Syncing device...");
+          log.info(device.getManufacturer());
+          log.info(device.getModel());
+          var foundDevice =
+              deviceTypeRepository.findDeviceTypeByManufacturerAndModel(
+                  device.getManufacturer(), device.getModel());
+
+          DeviceType deviceToSync =
+              foundDevice.orElseGet(
+                  () -> {
+                    // Have we seen this device before? Gets its specimens
+                    if (!deviceSpecimens.containsKey(
+                        device.getModel() + device.getManufacturer())) {
+
+                      deviceSpecimens.put(
+                          device.getModel() + device.getManufacturer(),
+                          getSpecimenTypeIdsFromDescription(device));
+                    }
+
+                    var specimensForDevice =
+                        deviceSpecimens.get(device.getModel() + device.getManufacturer());
+                    var supportedDisease =
+                        getSupportedDiseaseFromVendorAnalyte(device.getVendorAnalyteName());
+
+                    if (supportedDisease.isEmpty()) {
+                      // Skip this device
+                      log.info("Device does not contain required disease data");
+                      log.info("Skipping...");
+                      return null;
+                    }
+
+                    // Device name should be the same as the model name, unless a device
+                    // with that name already exists.
+                    // If so, prepend with the manufacturer name before creation
+                    var existingDeviceWithName =
+                        deviceTypeRepository.findDeviceTypeByName(device.getModel());
+                    var deviceName =
+                        existingDeviceWithName == null
+                            ? device.getModel()
+                            : String.join(" ", device.getManufacturer(), device.getModel());
```

Comment:
Just to make sure I'm understanding correctly - the issue is that we might have 2 devices with the same model names but different manufacturers? e.g., both manufacturers "HealthQuest Esoterics" and "P23 Labs" have the model "TaqPath SARS-CoV-2 Assay." 

In this case I think we should include the manufacturer name with both devices so the user can differentiate. Looping in @lauraykerr for her thoughts.

Reply:
@lauraykerr Actually, since we're including both manufacturer + model in the new device selection dropdown, do we need to include the manufacturer name in the device name field as well? I would think after adding the device to their facility they would just need the model name (assuming they aren't using both devices). 

One potential issue with the `Model` + (`manufacturer`) treatment is that the manufacturer name might get cut off in a dropdown. But, maybe we don't need this at all if we feel like the table is enough to differentiate?

Reply:
@lg-king That occurred to me too. I think the the table in the initial device selector should be enough and we don't need to include manufacturer after all. I do wonder if we should have some differentiator on the off chance a facility uses two different devices with the same name, but it doesn't seem like something we should prioritize designing for.